parseInt.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+    let element = document.querySelector('REPLACE_WITH_CSS_SELECTOR');
+
+    if (element) {
+        let elementText = element.innerText;
+        let parsed = parseInt(elementText);
+        
+        if (isNaN(parsed)) {
+         throw Error('Inner text does not contain an integer');
```

Comment:
This isn't quite accurate. "Contains" often means "includes somewhere inside it" so you probably mean "is not an integer".

Reply:
That is what I was trying to get at @srvance. The workflow here is extracting inner text, parsing that text for integers, and if there are no integers present in the inner text then it will throw the NaN. That if statement serves to catch that NaN if it is thrown which tells me that the inner text that was extracted does not contain an integer. Maybe we could talk about how I can improve this so that the intention behind the code is clear?